packages/harmony/src/foundations/color/Color.stories.mdx

```diff
@@ -0,0 +1,193 @@
+    surface1: 'Use for background surface',
+    surface2: 'Use for background surface emphasis'
+  }}
+/>
+
+### Border
+
+Semantic border variables are applied to the exterior stroke of various elements. The strong variable should be applied when the stroke in on a colored background.
+
+<ColorPaletteTable
+  colorType='border'
+  descriptions={{
+    default: 'Use for default border color',
+    strong: 'Use for border on color background'
+  }}
+/>
+
+### Focus
+
+Focus is used frequently as a secondary action color for things like selectable pills, links, radio buttons, inputs, etc. It should be applied to all active or focused states for components.
+
+<ColorPaletteTable
+  colorType='focus'
+  descriptions={{
+    focus: 'Use for focus states'
+  }}
+/>
+
+### Status
+
+Status colors are used to display status states: error, warning, and success.
+
+<ColorPaletteTable
+  colorType='status'
+  descriptions={{
+    error: 'Use for error states',
+    warning: 'Use for warning states',
+    success: 'Use for success states'
+  }}
+/>
+
+## Usage and Examples
+
+- Semantic variables should be used throughout your design. Avoid using primitives and #HEX values when possible.
+- When text and icons are side-by-side they should have the same semantic color variable applied to them.
+- **Border-strong** should be applied to elements on colored backgrounds.
+- Focus should be used as the default for interactive elements.
+- #FFFFFF should not exist in your designs.
+
+### How to use background and "white"
+
+- Use **background-default** to achieve our default background color in all modes.
+- Use **background-white** when applying white to a card or container.
+- Use **static-white** when you want white to remain white in all modes. (ex: ✓ in verified badge)
+
+export function BackgroundColorExample() {
+  const theme = useTheme()
+  const { color, cornerRadius, spacing, shadows } = theme
+
+return (
+
+    <Flex
+      flex={1}
+      p='2xl'
+      css={{
+        background: color.background.default,
+        border: `1px solid ${color.border.strong}`,
+        borderRadius: cornerRadius.m
+      }}
+    >
+      <Flex
+        flex={1}
+        alignItems='center'
+        justifyContent='center'
+        css={{
+          height: spacing.unit20,
+          background: color.background.white,
+          boxShadow: shadows.mid,
+          borderRadius: cornerRadius.m
+        }}
+      >
+        <Text color='default'>
+          Display Name <IconVerified size='xs' /> <IconTokenSilver size='xs' />
+        </Text>
+      </Flex>
+    </Flex>
+
+)
+}
+
+<Unstyled>
+  <Flex gap='3xl'>
+    <ThemeProvider theme='day'>
+      <BackgroundColorExample />
+    </ThemeProvider>
+    <ThemeProvider theme='dark'>
+      <BackgroundColorExample />
+    </ThemeProvider>
+    <ThemeProvider theme='matrix'>
+      <BackgroundColorExample />
+    </ThemeProvider>
+
+  </Flex>
+</Unstyled>
+
+## Related
+
+{/* TODO <RelatedComponents /> */}
```

Comment:
still todo?

Reply:
and if not needed yet, we could probably just comment out the title until this is added

Reply:
will do, yeah we need the other foundation docs for this to be relevant